src/Form/AdminSettings.php

```diff
@@ -155,13 +155,28 @@ public function buildForm(array $form, FormStateInterface $form_state, Request $
       ],
       '#description' => $this->t('<dl><dt>Never</dt><dd>Never collapse ad slots.</dd><dt>Collapse only</dt><dd>Collapse before any ad is loaded. Useful if ad slots will get filled most of the time.</dd><dt>Expand only</dt><dd>Collapse all divs on the page before any ads are fetched and expand if an ad is loaded into the ad slot. Useful if ad slots will stay empty most of the time.</dd></dl>'),
     ];
+    $form['global_display_options']['slug_position'] = [
+      '#type' => 'radios',
+      '#title' => $this->t('Display slug'),
+      '#default_value' => $config->get('slug_position'),
```

Comment:
When I visit this the admin form, both radios are empty, we need to provide the default value and select 'Inside the ad container' if this setting is empty